feelslikemonday/app/src/main/java/com/example/feelslikemonday/model/followeeMoodEvent.java

```diff
@@ -0,0 +1,29 @@
+package com.example.feelslikemonday.model;
+
+/**
+ * This is a model class for every user that your follow
+ * It keeps track of their username and most recent Mood event
+ */
+public class followeeMoodEvent {
```

Comment:
Class naming convention should have this as FolloweeMoodEvent

Reply:
ok, done